docs/docs/clients/rest.md

```diff
@@ -305,3 +305,51 @@ for identity in page_of_identities.json()['results']:
     identity_data = session.get(f"{IDENTITY_URL}")
     print(identity_data.json())
 ```
+
+### Delete an Edge Identity
+
+```python
+import os
+
+import requests
+
+TOKEN = os.environ.get("API_TOKEN")  # obtained from Account section in dashboard
+ENV_KEY = os.environ.get("ENV_KEY")  # obtained from Environment settings in dashboard
+IDENTITY_UUDI = os.environ["IDENTITY_UUDI"] # must (currently) be obtained by inspecting the request to /api/v1/environments/{ENV_KEY}/edge-identities/{IDENTITY_UUDI} in the network console
+BASE_URL = "https://api.flagsmith.com/api/v1"  # update this if self hosting
```

Comment:
this should be edge api?

Reply:
Yes. It is referring to the edge api as you can see in line 327 https://github.com/Flagsmith/flagsmith/blob/cfa760141ae6c896a29f978cd059bf2742924093/docs/docs/clients/rest.md#L327. As far as I understand the base URL is the same for edge and non-edge API calls, right?

Reply:
Sorry, my bad, I changed the base URL to point edge.api.flagsmith.com